user-operator/src/main/java/io/strimzi/operator/user/model/acl/SimpleAclRule.java

```diff
@@ -118,6 +121,118 @@ public String toString() {
                 "operation: " + operation + ")";
     }
 
+    public AccessControlEntry toKafkaAccessControlEntry(KafkaPrincipal principal)   {
+        AclPermissionType kafkaType;
```

Comment:
Do we stil need to have our own `AclPermissionType`, since it seems to have a subset of the cases of `type`?

Reply:
Maybe I didn't get what you mean but the `type` is a `AclRuleType` from our API model which is used in how we describe the ACLs in our `KafkaUser` resource. Here we are mapping this one to the Kafka provided `AclPermissionType` which is necessary for creating the Kafka objects (see `AccessControlEntry`) used by the Admin Client.

Reply:
Ah, OK! I didn't notice that it was from our api model (I thought it might be something internal to the UO). That's OK then.